.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,42 @@
+name: Check stale issues
+
+on:
+  workflow_dispatch: # FIXME: remove
+  schedule:
+    - cron: '0 12 * * *'
+
+env:
+  PERIOD: 14 # in days
+  WAIT_FOR_INFO: "waiting for more input"
+  DRY_RUN: true
+
+jobs:
+  check-issue-status:
+    name: Retrieve all open issues that are inactive
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get inactive open issues
+        id: stale_issues
+        uses: ./.github/actions/stale_issues
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          period: ${{ env.PERIOD }}
+      - name: Comment on found issues
+        if: ${{ success() }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          for issue_url in ${{ steps.stale_issues.outputs.result }}; do
+            # Filter stale issues that were triaged and are not waiting for more information
+            stale_triaged=$(gh issue view $issue_url \
+                                    --json labels \
+                                    --jq '.labels | map(.name != "$WAIT_FOR_INFO") | all and length > 0')
```

Comment:
I think that label filtering should be a part of `action.yml`.

Reply:
When I did that, the idea was to have one action that returns a list of stale issues, and then you could apply any rule you want on the result, for instance, filter them by the label. But yeah, I can move it to the action.